plugins/chat/assets/stylesheets/common/chat-navbar.scss

```diff
@@ -38,8 +38,10 @@
     @include ellipsis();
     font-weight: 700;
 
```

Comment:
Is it possible these are wrong additions? They seem to cause misalignment on desktop now for the threads tab and have no effect for the other routes 
<img width="676" alt="image" src="https://github.com/discourse/discourse/assets/101828855/5993d2a1-b5b7-4ecd-b1bc-2dadbaf40043">